packages/@aws-cdk/aws-codepipeline-actions/lib/codecommit/source-action.ts

```diff
@@ -68,6 +68,7 @@ export class CodeCommitSourceAction extends Action {
 
     super({
       ...props,
+      resource: props.repository,
```

Comment:
Is this what makes cross-stack magic work?

Reply:
Yep.